tests/ocpp_tests/README.md

```diff
@@ -54,7 +62,7 @@ specified in `test_sets/ocpp201/remote_control.py`.
 If you run the test cases individually,
 make sure to have all required certificates and configs
 for the test cases installed using the
-convenience scripts inside [test_sets/everest/aux](test_sets/everest-aux/)
+convenience scripts inside [test_sets/everest-aux](test_sets/everest-aux/)
```

Comment:
Isn't the everest-aux directory in another repo (everest-framework or something)? I could not find it in the test_sets directory of everest-core.

Reply:
Its here: https://github.com/EVerest/everest-core/tree/feature/ocpp-integration-tests/tests/ocpp_tests/test_sets/everest-aux

Reply:
it's even in the pull request :D. Oops.

Might have searched for everest_aux or something indeed.